app/models/landing.rb

```diff
@@ -56,4 +58,12 @@ def to_s
   def to_param
     slug
   end
+
+  private
+
+  def set_emphasis
+    if emphasis
+      Landing.where.not(id: id).update_all(content: { emphasis: false })
```

Comment:
Attention, ça efface tout le reste de `content`! A priori, on peut faire `update_all(emphasis: false)`, non?

Reply:
ah oui pardon

Reply:
ah oui, forcément, update_all c’est l’accès direct en base, pardon. C’était `update` tout court (sur une query).

(https://scottbartell.com/2020/01/30/set-attributes-in-active-record-rails-6/ dans les bookmarks)

Reply:
je sauvegarde ce lien direct